azure-communication-ui/calling/src/main/res/layout/azure_communication_ui_calling_listview.xml

```diff
@@ -3,11 +3,18 @@
    Licensed under the MIT License.
   -->
 
-<androidx.recyclerview.widget.RecyclerView xmlns:android="http://schemas.android.com/apk/res/android"
-    android:id="@+id/bottom_drawer_table"
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/audio_drawer_container"
```

Comment:
is name right?

Maybe +id/bottom_drawer

if it contains id/bottom_drawer_table